### DISCUSSION TITLE
WiFi signal in dBm

### DISCUSSION THREAD

**Discussion by Mike86-CZE** [PE5PVB/TEF6686_ESP32#610] [Ideas]
Since the portable TEF does not have an external wifi antenna, the signal is always very weak.  So the graphical representation of the signal strength is not very useful.  I would like to optionally display the wifi strength in dBm on the main screen.

**Comment by PE5PVB** [comment]
Problem is that the dBm value on WiFi changes continiously few times per second. It's almost impossible to do this.
